core/ui/ControlPanel/Saving/GitHub.tid

```diff
@@ -0,0 +1,16 @@
+title: $:/core/ui/ControlPanel/Saving/GitHub
+tags: $:/tags/ControlPanel/Saving
+caption: {{$:/language/ControlPanel/Saving/GitService/GitHub/Caption}}
```

Comment:
It doesn't seems to possible to use macros in caption (`<<service-name>>`), but if possible, there is no need for separate captions.